GraknOptions.java

```diff
@@ -23,13 +23,28 @@
 
 import java.util.Optional;
 
-import static grakn.client.common.exception.ErrorMessage.Client.NEGATIVE_BATCH_SIZE;
+import static grakn.client.common.exception.ErrorMessage.Client.NEGATIVE_VALUE_NOT_ALLOWED;
```

Comment:
Just a note - I think we had this specific message because we ideally want each error message to refer to a specific error. But honestly, I think this may need a broader rethink so this is fine for now.